content/docs/configuration/acme/dns01/akamai.md

```diff
@@ -17,7 +17,7 @@ kind: Secret
 metadata:
   name: akamai-secret
 type: Opaque
-stringData:
+data:
```

Comment:
I don't agree with this change. 
`stringData` is the correct way to supply these credentials when creating the Secret because it otherwise the user would have to manually base64 encode the values.
See:

 * https://kubernetes.io/docs/concepts/configuration/secret/#restriction-names-data

Reply:
Ok makes sense